app/router.ts

```diff
@@ -54,8 +54,6 @@ Router.map(function() {
         this.mount('registries', { path: '--registries' });
     }
 
-    this.route('guid-file', { path: '--file/:guid' });
-
```

Comment:
This needs to stay as well, otherwise we won't be able to view files.

Reply:
This is as well.